ui/src/templates/utils/index.test.ts

```diff
@@ -49,43 +49,49 @@ describe('Templates utils', () => {
 })
 
 describe('the Community Template url utilities', () => {
-  it('returns the template name and extension from an arbitrary url', () => {
-    const {name, extension} = getTemplateNameFromUrl(
+  it('returns the template name, directory, and extension from an arbitrary url', () => {
+    const {name, extension, directory} = getTemplateNameFromUrl(
       'https://github.com/influxdata/influxdb/blob/master/pkger/testdata/dashboard_params.yml'
     )
+
     expect(name).toBe('dashboard_params')
     expect(extension).toBe('yml')
+    expect(directory).toBe('testdata')
   })
 
   it('returns the template name and extension from the official community templates github repo', () => {
-    const {name, extension} = getTemplateNameFromUrl(
+    const {name, extension, directory} = getTemplateNameFromUrl(
       'https://github.com/influxdata/community-templates/blob/master/csgo/csgo.yml'
     )
     expect(name).toBe('csgo')
     expect(extension).toBe('yml')
+    expect(directory).toBe('csgo')
```

Comment:
```suggestion
    expect(directory).toBe('hooray')
```

just because it's good to change things up to make sure tests still work